modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -221,25 +221,23 @@ Salt-managed production configuration supports all features supported by all maj
 
 You can configure {productname} to automatically accept the keys of newly deployed terminals.
 This is achieved using Salt grains.
-For more information about automatically accepting Salt grains, see https://docs.saltstack.com/en/latest/topics/tutorials/autoaccept_grains.html.
 
 [IMPORTANT]
 ====
 Automatically accepting keys is less secure than manually checking and accepting keys.
 Only use this method on trusted networks.
 ====
 
+There are three different ways you can configure auto-signed grains:
 
-To do this, you first need to configure Saltboot, then configure the {productname} Server.
+* Configure Saltboot to send automatically signed grains once and then delete them.
```

Comment:
Don't we need to indent the 2nd sentence of every list item?  2 spaces?

Reply:
Yes, you're right.